ios/BurntModule.swift

```diff
@@ -235,34 +235,34 @@ public class BurntModule: Module {
     }.runOnQueue(.main)
     
     AsyncFunction("alertAsync")  { (options: AlertOptions) -> Void in
-      var preset: SPAlertIconPreset?
+      var preset: AlertIcon?
       do {
-        preset = try options.preset.toSPAlertIconPreset(options)
+        preset = try options.preset.toAlertIcon(options)
       } catch {
         log.error("Burnt Alert error: \(error)")
       }
+
+      guard let window = UIApplication.shared.windows.filter({ $0.isKeyWindow }).first else { return }
       
-      let view = SPAlertView(
+      let view = AlertAppleMusic16View(
```

Comment:
`AlertAppleMusic16View` corresponds to `SPAlertView`.
AlertKit also supports `AlertAppleMusic17View`, but should we expose an ios-only prop for that?